tasks/engine_setup.yml

```diff
@@ -0,0 +1,65 @@
+---
+- block:
+  - name: Set answer file path
+    set_fact:
+      answer_file_path: "/tmp/answerfile-{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}.txt"
+
+
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: "{{ answer_file_path }}"
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is undefined
+
+  - name: Copy custom answer file
+    template:
+      src: "{{ ovirt_engine_setup_answer_file_path }}"
+      dest: "{{ answer_file_path }}"
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is defined
+
+  - name: Update setup packages
+    yum:
+      name: "ovirt*setup*"
+      state: latest
+    when: ovirt_engine_setup_update_setup_packages
+
+  - name: Update all packages
+    yum:
+      name: "*"
+      state: latest
+    when: ovirt_engine_setup_update_all_packages
+
+  - name: Set accept defaults parameter if variable is set
+    set_fact:
+      accept_defaults: "{{ '--accept-defaults' if ovirt_engine_setup_accept_defaults else '' }}"
+
+  - name: Run engine-setup with answerfile
+    command: "engine-setup --config-append={{ answer_file_path }} {{ accept_defaults }}"
+    tags:
+      - skip_ansible_lint
+
+  - name: Make sure `ovirt-engine` service is running
+    service:
+      name: ovirt-engine
+      state: started
+
+  - name: Check if Engine health page is up
+    uri:
+      url: "http://{{ ansible_fqdn }}/ovirt-engine/services/health"
```

Comment:
The health service is considered deprecated and might be removed in the future, see also: https://bugzilla.redhat.com/show_bug.cgi?id=1027210

If you decide to keep the code as-is, please open an issue to replace this and comment on that bug - so that we know that ovirt-ansible-engine-setup uses it too.

Reply:
Ok, I'll try to replace and make it point to the api